tensorboard/compat/tensorflow_stub/io/gfile.py

```diff
@@ -220,6 +220,7 @@ class S3FileSystem(object):
     def __init__(self):
         if not boto3:
             raise ImportError("boto3 must be installed for S3 support.")
+        self.s3_endpoint = os.environ.get("S3_ENDPOINT", None)
```

Comment:
Please prefix with an underscore (`self._s3_endpoint`).

Reply:
@nfelt i made the changes as you suggested please look into this, i ran black and reformatted the code but still the lint error exists!

Reply:
Can you rerun black?  As long as you're running it correctly it should ensure the lint passes on CI.

It looks like the problem is the spaces around the equals sign in the `endpoint_url = self._s3_endpoint` line below.